lib/_http_incoming.js

```diff
@@ -60,25 +59,21 @@ function IncomingMessage(socket) {
 }
 util.inherits(IncomingMessage, Stream.Readable);
 
-
 exports.IncomingMessage = IncomingMessage;
 
-
 IncomingMessage.prototype.setTimeout = function(msecs, callback) {
   if (callback)
     this.on('timeout', callback);
   this.socket.setTimeout(msecs);
 };
 
-
 IncomingMessage.prototype.read = function(n) {
   this._consuming = true;
   this.read = Stream.Readable.prototype.read;
   return this.read(n);
 };
 
-
-IncomingMessage.prototype._read = function(n) {
+IncomingMessage.prototype._read = function() {
```

Comment:
Unfortunately omitting unused arguments in this fashion has the side-effect of making V8 use an `ArgumentsAdaptor`. I'm not sure what the best route to fix this is – either we enable unused vars, or we explictly "use" unused vars in the function body in no-op statements.